### PR TITLE
feat(payment): INT-4242 Add Quadpay payment method

### PR DESCRIPTION
**Pull request by mauricio-sg**
## What? [INT-4242](https://jira.bigcommerce.com/browse/INT-4242)
Add **Quadpay** payment method.

## Why?
To make **Quadpay** available at checkout.

## Testing / Proof
Quadpay

## Sibling PR
This PR depends on https://github.com/bigcommerce/checkout-sdk-js/pull/1129 to pay.
This PR depends on https://github.com/bigcommerce/bigcommerce/pull/40762 to display the Quadpay logo.

@bigcommerce/apex-integrations  @bigcommerce/checkout

### CODE REVIEW

**Review by cmedinilla — APPROVED** [review]
LGTM!

File: src/app/payment/paymentMethod/BlueSnapV2PaymentMethod.spec.tsx [hunk 1]
```diff
@@ -122,6 +122,32 @@ describe('when using BlueSnapV2 payment', () => {
             .toHaveLength(1);
     });
 
+    it('renders modal but does not append bluesnap payment page because is empty', async () => {
```
Comment (capsula4) [inline src/app/payment/paymentMethod/BlueSnapV2PaymentMethod.spec.tsx:125]: was this change intended?

Comment (mauricio-sg) [reply src/app/payment/paymentMethod/BlueSnapV2PaymentMethod.spec.tsx]: Yes, it was intended because yesterday I pulled from master and global coverage threshold for branches was not met and circleci was complaining. But now seems to be working again.

Comment (mauricio-sg) [reply src/app/payment/paymentMethod/BlueSnapV2PaymentMethod.spec.tsx]: I opened up a new PR: https://github.com/bigcommerce/checkout-js/pull/600

**Review by capsula4 — APPROVED** [review]

**Review by TomA-R — APPROVED** [review]

**Comment by JoseBobadilla20** [comment]
💚 Tested 
QA Details: I tested Quadpay in the US with USD and CA with CAD
